templates/geosafe/analysis/modal/impact_card.html

```diff
@@ -23,7 +23,7 @@ <h4 class="modal-title" id="impact_card_label">Analysis Result</h4>
                 <div class="container col-xs-12">
                     <div class="row">
                         <div class="impact-thumbnail col-xs-6">
-                            <img id="impact-layer"/>
+                            <img src="{{ analysis.impact_layer.get_thumbnail_url }}"/>
```

Comment:
We just use the thumbnail url directly. This is possible because:

1. This template (impact_card.html), is generated to show analysis layer. So it is guaranted that an impact layer will exist. Because if the analysis failed, this impact summary will not show up.
2. `analysis` object were provided in the Jinja2 context. So we can just access the object.